printer_mods/KevinAkaSam/VEFACH/README.md

```diff
@@ -0,0 +1,40 @@
+
+<img src="https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/CAD3.png" width="400"/> <img src="https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/CAD4.png" width="400"/>
+
+### Why?
+Since the printer produces many small particles, the goal is to filter out as many of them as possible. A few months ago my friend Deutherius had the idea to equip the original exhaust fan with activated coal and a Hepa filter, so I designed this filter inserts. The activated carbon removes e.g. the smell of ABS and the Hepa filter catches most of the small particles.
+ 
+### How does it work?
+The air first blows through activated carbon particles and then through the Hepa filter. This still doesn't remove all particles, but it is better than nothing. This mod is fully compatible with the stock exhaust fan housing, except original M3x30mm screws that hold the fan are too long and need to be replaced with M3x25mm. The filter materials can be removed and replaced very easily via the original cover.
+
+### How to print?
+You just need to print the two inserts for the housing. The infill is used as mesh for the activated carbon container. I have attached a few screenshots of how to prepare the print with the dummy filter file. The green options shown in the last two screenshots are optional. Either you set the infill of the whole bucket higher or you add another setting for the dummy FIlter. Except for the top end, the print consists of walls only.
+
+
+1: Select Filter Dummy as modifier        |  2: Select settings for modifier
+:-------------------------:|:-------------------------:
+![](https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/1.jpg)  |  ![](https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/2.jpg)
+3: Select Perimeters Shell        |  4: Set Top-/Bottom-Layer to 0
+![](https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/3.jpg)  |  ![](https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/4.jpg)
+4: Set infill density + Type to Grid        |  5: (Optional) Set different infill density for modifier
+![](https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/5.jpg)  |  ![](https://github.com/kevinakasam/VoronUsers/blob/master/printer_mods/KevinAkaSam/VEFACH/Images/6.jpg)
+
+Make sure that the dummy filter and the bucket are centered. If you want to move them, then select both at the same time so you don't change the cropping area.
+You should choose the infill density so that the carbon particles can not fall out, but also not too high, so as not to close the airflow too much. Please test before printing. The settings given are for demonstration purposes only!
+
+### What do I need?
+4x M3x25mm screws
+
+1x 40mmx80mm Hepa Filter (https://a.aliexpress.com/_mNsj3qz)
```

Comment:
Please replace this with a normal (non-redirect) link, looks like it's just `https://www.aliexpress.com/item/1005001615722382.html`